packages/client/wallets/aa/src/blockchain/BlockchainNetworks.ts

```diff
@@ -0,0 +1,87 @@
+export const EVMBlockchain = {
```

Comment:
This class should be in the core package, imported by wallets. but the team can do this on another PR

Reply:
yeah definitely, we should actually have a package like `@crossmint/common-sdk-base/core` wdyt?

Reply:
sg